### ISSUE TITLE
containerClasses: override the defaults?

### ISSUE THREAD

**Issue by dloused** [acidjazz/tv-toast#16]
```
{
  modules: [
    [ 'nuxt-tailvue', { all: true, toast: { defaults: { containerClasses: [ 'mt-12' ] } } } ],
  ]
}
```

As per example, container classes only **extend** the default class? Is it possible to override the defaults ('z-40', 'fixed', 'inset-0', 'flex', 'flex-col-reverse', 'items-end', 'justify-center',
  'px-4', 'py-6', 'pointer-events-none', 'sm:p-6', 'sm:items-end', 'sm:justify-end') in the module params?

**Comment by acidjazz** [comment]
@dloused have  you tried doing this by just adding classes that cancel the default ones out?

**Comment by dloused** [comment]
Yes i tried that, for eg. i want the toast at the bottom. By adding sm:justify-start, the class output is "sm:justify-end sm:justify-start", with the 1st taking precedence

**Comment by acidjazz** [comment]
@dloused I thought css classes later in the order took precedence?

**Comment by acidjazz** [comment]
@dloused I see your point, well maybe we can do an `override:` that you can use instead and start with a clean slate?

**Comment by dloused** [comment]
That would be a good option.!

**Comment by acidjazz** [comment]
@dloused i added to my open source TODO, feel free to send a PR if you want it sooner!
